.gitignore

```diff
@@ -10,3 +10,35 @@ __pycache__
 .python-version
 *.egg-info
 dist
+
+## Log exclusion
+*.log
+
+## Report Output Exclusion
+*.pdf
+
+## Data input exclusion
+*.json
+
+
+## Virtualenv
+.venv
+.vs
+
+## VS Code
+.vscode
+
+.profile-data
+
+.env
+old.env
+/.vsenv
+
+*.mmdb
+.geoipupdate.lock
+config.ini
+Dockerfile-x
+*.swp
+
+*.DS_store
+*.vscode
```

Comment:
Please clean up this .gitignore file:
- Ensure all section headers are formatted consistently with `##` before and after the section name
- Alphabetize the sections
- Alphabetize (and remove blank lines) within each section
- Ensure that the contents of each section are indeed correct (e.g. `.geoipupdate.lock` doesn't belong in the "VS Code" section)

Reply:
Resolved in 79edc2f944cfa827bc0e03060e72364d2a17a9bc  Updated section headers, cleaned up duplicates, and ordered alphabetically.